pkg/cmd/templates/templates.go

```diff
@@ -49,7 +51,8 @@ Examples:
 {{end}}{{end}}{{ if or $visibleFlags.HasFlags $explicitlyExposedFlags.HasFlags}}
 Options:
 {{ if $visibleFlags.HasFlags}}{{flagsUsages $visibleFlags}}{{end}}{{ if $explicitlyExposedFlags.HasFlags}}{{flagsUsages $explicitlyExposedFlags}}{{end}}{{end}}{{ if .HasSubCommands }}
-Use "{{$rootCmd}} <command> --help" for more information about a given command.{{end}}{{ if $optionsCmdFor}}
+Use "{{$rootCmd}} <command> --help" for more information about a given command.{{end}}
+{{ if $optionsCmdFor}}{{ if $hasContainerOption}}Use "{{$describeCmdFor}} POD" to obtain container name values for that pod.{{end}}
```

Comment:
Thanks! I went ahead and added the linebreak after the `if` statement for `$hasContainerOption` so that it still adds a linebreak in cases like `oc exec -h`, but not extra lines elsewhere